shell/profiling/sampling_profiler.h

```diff
@@ -29,15 +29,29 @@ struct CpuUsageInfo {
   double total_cpu_usage;
 };
 
+/**
+ * @brief Memory usage stats. `dirty_memory_usage` is the number of mega bytes
+ * such that the app uses its physical memory for dirty memory. Dirty memory
+ * is the memory data that cannot be paged to disk. `owned_shared_memory_usage`
+ * is the number of mega bytes such that the app uses its physicaal memory for
+ * shared memory, including loaded frameworks and executables. On iOS, it's
+ * `physical memory - dirty memory`.
+ */
+struct MemoryUsageInfo {
+  double dirty_memory_usage;
```

Comment:
Oh, I missed the "mega bytes" in the struct documentation... Maybe also put a capitalized MB there, or add a document for each field so they're more prominent.

Reply:
Done.

Reply:
I rephrased the docstring a bit so that "MB" should be more obvious than "mega bytes". I will keep the docstring as is so it is consistent with the CPU part.

As another way, we can also rename them to `dirty_memory_usage_mb` and `owned_shared_memory_usage_mb`, WDYT?

Reply:
The `MB` docstring as it is now looks good!